front50-gcs/src/main/java/com/netflix/spinnaker/front50/config/GcsConfig.java

```diff
@@ -166,6 +169,11 @@ public PipelineDAO pipelineDAO(GcsStorageService service, Registry registry) {
   }
 
   @Bean
+  public PipelineTemplateDAO pipelineTemplateDAO(StorageService storageService, Registry registry) {
+    return new DefaultPipelineTemplateDAO(storageService, Schedulers.from(Executors.newFixedThreadPool(25)), PIPELINE_TEMPLATE_REFRESH_MS, registry);
```

Comment:
Perhaps too many threads (depends on how many # of pipeline templates and concurrent calls to getByScope) but I've got a PR in flight that lets this be a little more configurable.